lego-mindstorms-pieces.py

```diff
@@ -161,10 +164,40 @@ def missing(omitted_set, datafile):
     print(','.join(order_list))
 
 
+def is_electric_part(wanted_part_no, print_set_link=False):
+    """
+    Detect if part is in EV3 Electric parts file, print link to standalone Set if wanted
+
+    return boolean
+    """
+
+    wanted_part_no = str(wanted_part_no)
+    datafile = os.path.join(SCRIPT_PATH, 'raw-data', 'Electric-parts.csv')
+
+    exit_code = False
+
+    with open(datafile) as f:
+        data_lines = f.readlines()[1:]
+        for line in data_lines:
+            (partno, legoid, legoshop_set) = line.split('\t')
```

Comment:
Oh, yet another one: The brackets are strictly not needed. For unpacking you can write:
```python
foo, bar, baz = my_triple_value
```

Reply:
Indeed, it work without brackets.
I'd grabbed it from your code in missing() and parse() functions ;-)